lib/msal-browser/src/utils/PopupUtils.ts

```diff
@@ -9,6 +9,16 @@ import { BrowserAuthError } from "../error/BrowserAuthError";
 import { AuthorizationUrlRequest } from "../request/AuthorizationUrlRequest";
 import { BrowserConstants, InteractionType, TemporaryCacheKeys } from "./BrowserConstants";
 
+/**
+ * Popup configurations for setting dimensions and position of popup window
+ */
+export type PopupConfiguration = {
```

Comment:
Can we call this something like `PopupDimensions` or `PopupWindowAttributes` instead? We should try to avoid creating confusion about whether this references popup api "configuration" or popup window "configuration" or both

Reply:
I originally considered dimensions but wondered if that would encompass position well. I changed it to `PopupWindowAttributes`, thanks for the suggestion.